bitnami/jupyterhub/values.yaml

```diff
@@ -1371,13 +1466,18 @@ singleuser:
     ## If not set and create is true, a name is generated using the fullname template
     ##
     name: ""
+    ## @param singleuser.serviceAccount.automountServiceAccountToken Allows auto mount of ServiceAccountToken on the serviceAccount created
+    ## Can be set to false if pods using this serviceAccount do not need to use K8s API
+    ##
+    automountServiceAccountToken: true
+    ## @param singleuser.serviceAccount.annotations Additional custom annotations for the ServiceAccount
+    ##
+    annotations: {}
 
   ## @section Single User Persistence parameters
-  ##
-
   ## Enable persistence using Persistent Volume Claims
   ## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
-  ##
+
   persistence:
     ## @param singleuser.persistence.enabled Enable persistent volume creation on Single User instances
     ## If true, use a Persistent Volume Claim, If false, use emptyDir
```

Comment:
I was about to mention that `singleuser.persistence.annotations`, but are these values even used anywhere?

Reply:
Yes, they're used. Check all the references in the `hub.configuration` parameter to these values. And I don't think that the annotations apply in this specific case since we're using these parameters to configure the Hub.

Reply:
Sorry, I was looking inside the `templates` and missed the references inside `values.yaml`.
Yes, I agree that `persistence.annotations` may not apply in this case